README.md

```diff
@@ -342,13 +342,51 @@ The osplaybookgenerator created above will automatically generate playbooks any
 
 9) Login to the 'openstackclient' pod and deploy the OSP software via the rendered ansible playbooks. At this point all baremetal and virtualmachine resources have been provisioned within the OCP cluster.
 
+The `tripleo-deploy.sh` script supports three actions:
+* `-d` - show the `git diff` of the playbooks to the previous accepted
+* `-a` - accept the new available rendered playbooks and tag them as `latest`
+* `-p` - run the ansible driven OpenStack deployment
+
+a) check for new version of renered playbooks and accept them
+
+```bash
+oc rsh openstackclient
+bash
+cd /home/cloud-admin
+
+# (optional) show the `git diff` of the playbooks to the previous accepted
+./tripleo-deploy.sh -d
+
+# (optional) accept the new available rendered playbooks (if available) and tag them as `latest`
```

Comment:
this actually isn't optional. If "new" playbooks have been generated and pushed to the repo you have to run with -a to accept them.

Reply:
correct, was wrong c/p